e2e/ci_tests_test.go

```diff
@@ -77,5 +77,30 @@ var _ = Describe("E2E TEST:"+checks.CheckCITests, func() {
 			Expect(scut.ValidateTestReturn(nil, "CI tests run", &expected, &result, &dl)).Should(BeTrue())
 			Expect(repoClient.Close()).Should(BeNil())
 		})
+		It("Should return absence of CI tests in a repo with unsquashed merges", func() {
```

Comment:
Want to clarify my understanding here - so if a repo has unsquashed merges, even if the repo runs CI tests before merging PRs, Scorecard will return absence of CI tests?

Reply:
In this case, there were CI tests run only on the merge commit of an unsquashed batch of commits (the merge commit is created automatically by Github after the PR is accepted and merged). If CI tests are run before the merge (i.e. they are run on any commit before the merge commit), scorecard would say that CI tests are present